src/util.rs

```diff
@@ -17,14 +17,8 @@ use crate::errors::*;
 ///
 /// Automatically append an extension if not presented.
 pub fn find_file(file: &str, subdir: Option<&str>, extension: Option<&str>) -> Option<PathBuf> {
-    // Set (or update) the extension
-    let mut file = PathBuf::from(file);
-    if let Some(extension) = extension {
-        file.set_extension(extension);
-    }
-
-    // Try full path
-    if file.exists() {
+    let file = PathBuf::from(file);
+    if let Some(file) = exists_with_extension(&file, extension) {
```

Comment:
I think it's okay to require full paths to include the extension, so this can be just `file.exists()`.

Reply:
That makes sense.